mula/scheduler/connectors/listeners/listeners.py

```diff
@@ -51,27 +58,26 @@ def __init__(self, dsn: str):
         super().__init__()
         self.dsn = dsn
 
+        self.connection = pika.BlockingConnection(pika.URLParameters(self.dsn))
+        self.channel = self.connection.channel()
```

Comment:
Just opened the docs to read that opening a channel here indeed is a good idea. Let's do that for the other services as well!